webpack/base.js

```diff
@@ -5,11 +5,11 @@ module.exports = () => {
   return {
     entry: {
       "login": [
-        "./src/apps/loyalty/containers/login/browser",
+        "babel-polyfill", "./src/apps/loyalty/containers/login/browser",
       ],
       "inquiries": [
-        "./src/apps/loyalty/containers/inquiries/browser",
-      ]
+        "babel-polyfill", "./src/apps/loyalty/containers/inquiries/browser",
```

Comment:
I forgot to mention that there was another issue where clicking wasn't working at all. There is no relation between the opacity bug and polyfill. The issue I saw seemed to come from inside React. We also use other functions like `Object.assign` that's not available on IE11, so we'd have to polyfill anyway.